proposals/0304-structured-concurrency.md

```diff
@@ -421,8 +421,8 @@ As already shown in the above examples, it is possible to call functions that in
 This is possible because of the existence of the `withUnsafeCurrentTask` API:
 
 ```swift
-static func withUnsafeCurrentTask<T>(
-    operation: (UnsafeCurrentTask?) throws -> T
+public func withUnsafeCurrentTask<T>(
+    body: (UnsafeCurrentTask?) throws -> T
```

Comment:
operation is nicer; the proposal is ahead of impl and better a bit, please don't change this